lighthouse-core/report/styles/report.css

```diff
@@ -790,6 +839,7 @@ body {
 
 .export-section {
   position: relative;
+  display: none;
```

Comment:
instead of hiding the export-button, hide the entire export-section when not in viewer context. So the dropdown won't cause the report page to be wider than it should be, which could make the report page scroll-able on x direction.